src/Okta.Sdk/Generated/IAppUser.Generated.cs

```diff
@@ -21,7 +21,7 @@ public partial interface IAppUser : IResource
 
         string ExternalId { get; }
 
-        string Id { get; set; }
+        string Id { get; }
```

Comment:
This should be mutable since devs can set it when calling AssignUserToApplication. Do you guys think this could be a problem?
cc @royalchan-okta @gormanho-okta